docs/setup-cluster/checklists/postgresql.rst

```diff
@@ -123,6 +123,7 @@ Install PostgreSQL using ``apt`` or ``yum``
       postgres=# CREATE DATABASE determined;
       postgres=# CREATE USER determined WITH ENCRYPTED PASSWORD 'determined-password';
       postgres=# GRANT ALL PRIVILEGES ON DATABASE determined TO determined;
+      postgres=# GRANT ALL ON SCHEMA public TO determined;
```

Comment:
If you don't switch to the determined database, don't you need to qualify this grant?  Or switch first?

Reply:
ah, that's just me not paying enough attention